src/shipit_risk_assessment/VERSION

```diff
@@ -1 +1 @@
-1.0.0
+0.1.0
```

Comment:
👍 ok, i will add a note that i need to document that <1.0.0 are pre-production versions